### DISCUSSION TITLE
tlg-catalog/stable/graphs/pharmacokinetic/pkcg03

### DISCUSSION THREAD

**Discussion by giscus** [insightsengineering/tlg-catalog#130] [General]
# tlg-catalog/stable/graphs/pharmacokinetic/pkcg03

Plot of Mean PK Concentration Over Time by Cohort

https://insightsengineering.github.io/tlg-catalog/graphs/pharmacokinetic/pkcg03.html

**Comment by legrasv** [comment]
Dear all,

I realized that this plot does not follow the pRED and gRED guidelines:
In the legend, "N=" should be the number of patients per cohort, and not the number of points.

See
Example 10c: Side-by-side Mean Concentration-time in Linear &amp; Log Scale for PKCG03

**Reply by Melkiades** [reply]
we need to fix this then. let me make it an issue

**Reply by edelarua** [reply]
This is fixed by https://github.com/insightsengineering/tlg-catalog/pull/132.
